indexd/drs/blueprint.py

```diff
@@ -16,15 +16,54 @@ def get_drs_object(object_id):
     """
     Returns a specific DRSobject with object_id
     """
+    expand = True if flask.request.args.get("expand") == "true" else False
+
     ret = blueprint.index_driver.get(object_id)
 
-    return flask.jsonify(indexd_to_drs(ret)), 200
+    data = (
+        indexd_to_drs(ret, expand=False, list_drs=False)
+        if not flask.request.args.get("expand")
```

Comment:
I'm a bit confused on this logic - why is there a different function call if the `expand` param is not available?